quickjs.c

```diff
@@ -98,6 +98,29 @@ const char* JS_GetVersion(void) {
 #undef STRINFIGY_
 #undef STRINGIFY
 
+static inline JSValueConst *vc(JSValue *vals)
+{
+    return (JSValueConst *)vals;
+}
+
+static inline JSValue unsafe_unconst(JSValueConst v)
```

Comment:
why unsafe?

Reply:
Because if you're not careful, you double-free. It's a "reader beware" signal.

Reply:
👍